includes/class-migration.php

```diff
@@ -964,4 +965,62 @@ public static function migrate_followers_to_ap_actor_cpt() {
 
 		self::update_postmeta_key( '_activitypub_user_id', Followers::FOLLOWER_META_KEY );
 	}
+
+	/**
+	 * Update _activitypub_actor_json meta values to ensure they are properly slashed.
+	 *
+	 * @param int $batch_size Optional. Number of meta values to process per batch. Default 100.
+	 * @param int $offset     Optional. Number of meta values to skip. Default 0.
+	 *
+	 * @return array|null Array with batch size and offset if there are more meta values to process, null otherwise.
+	 */
+	public static function update_actor_json_storage( $batch_size = 100, $offset = 0 ) {
+		global $wpdb;
+
+		// phpcs:ignore WordPress.DB.DirectDatabaseQuery
+		$meta_values = $wpdb->get_results(
+			$wpdb->prepare(
+				"SELECT post_id, meta_value FROM {$wpdb->postmeta} WHERE meta_key = '_activitypub_actor_json' LIMIT %d OFFSET %d",
+				$batch_size,
+				$offset
+			)
+		);
+
+		foreach ( $meta_values as $meta ) {
+			// phpcs:ignore
+			$json = \json_decode( $meta->meta_value, true );
```

Comment:
```suggestion
			\json_decode( $meta->meta_value, true );
```
Is the `phpcs:ignore` flag to stop the unused variable warning? If so, let's not define it.